sport/app/AppLoader.scala

```diff
@@ -31,6 +32,9 @@ class AppLoader extends FrontendApplicationLoader {
 
 trait SportServices {
   def wsClient: WSClient
+  lazy val footballClient = wire[FootballClient]
+  lazy val competitionDefinitions = CompetitionsProvider.allCompetitions
+  lazy val competitionsService = wire[CompetitionsService]
```

Comment:
@alexduf I wanted to make the competitions definitions a parameter with a default value but macwire doesn't seem to support that